crypto/evp/evp_exchange.c

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
`crypto/evp/` contains EVP stuff...  why do we need to repeat the prefix in the file name?  Yeah, I know that *some* files are named that way, but far from all, and I fail to see why that type of tautology is needed, or even desirable.

Reply:
I renamed it to exchange.c